qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -0,0 +1,54 @@
+- driver_load_stress:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    suppress_exception = False
+    sleep_time = 300
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+    only Windows
+    variants:
+        - before_bg_test:
+            run_bg_flag = "before_bg_test"
+        - during_bg_test:
+            run_bg_flag = "during_bg_test"
+        - after_bg_test:
+            run_bg_flag = "after_bg_test"
+    variants:
+        - with_block:
+            sub_test = driver_load
+            driver_name = viostor
+            run_bgstress = iozone_windows
+            drive_format_image1 = ide
+            cdrom_cd1 = isos/windows/winutils.iso
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = virtio
+            force_create_image_stg = yes
+            iozone_cmd = "%s:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f %s:\testfile"
+            iozone_timeout = 600
+            wait_bg_time = 300
+            driver_id_pattern = "(.*?):.*?VirtIO SCSI Disk Device"
+            device_key = "VolumeName='Windows'"
+        - with_vioscsi:
+            sub_test = driver_load
+            driver_name = vioscsi
+            run_bgstress = iozone_windows
+            cd_format_fixed = ide
+            drive_format_image1 = ide
+            cdrom_cd1 = isos/windows/winutils.iso
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = scsi-hd
+            force_create_image_stg = yes
+            iozone_cmd = "%s:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f %s:\testfile"
```

Comment:
Where give the drive letter?

       Error using popen() on uname
        -M option suppressed.
        Command line used: /Iozone/iozone -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f :\testfile
        Output is in Kbytes/sec
        Time Resolution = 0.000012 seconds.
        Processor cache size set to 1024 Kbytes.
        Processor cache line size set to 32 bytes.
        File stride size set to 17 * record size.
                                                            random  random    bkwd   record   stride    
              KB  reclen   write rewrite    read    reread    read   write    read  rewrite     read   fwrite frewrite   fread  freread
         1048576      64cygwin warning:
  MS-DOS style path detected: :\testfile
  Preferred POSIX equivalent is: :/testfile
  CYGWIN environment variable option "nodosfilewarning" turns off this warning.
  Consult the user's guide for more details about POSIX paths:
    http://cygwin.com/cygwin-ug-net/using.html#using-pathnames

Can not open temp file: :\testfile
open: No such file or directory